### PR TITLE
[Adapter] Add support for Nighthawk

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 899176a8fa3a474e75212e5f20d83c146914f4b0 at: https://65d5940eec1505586e3c14e0--meshery-docs-preview.netlify.app

File: server/models/adapter.go [hunk 1]
```diff
@@ -8,18 +8,19 @@ import (
...
+	AppMesh   = Adapter{Name: "meshery-app-mesh", Location: "10005"}
+	Traefik   = Adapter{Name: "meshery-traefik-mesh", Location: "10006"}
+	Kuma      = Adapter{Name: "meshery-kuma", Location: "10007"}
+	Nginx     = Adapter{Name: "meshery-nginx-sm", Location: "10010"}
+	Cilium    = Adapter{Name: "meshery-cilium", Location: "10012"}
+	Nighthawk = Adapter{Name: "meshery-nighthawk", Location: "10013"}
 )
 
-var ListAvailableAdapters = []Adapter{Istio, Linkerd, Consul, Kuma, NSM, Nginx, AWS, Cilium, Traefik}
+var ListAvailableAdapters = []Adapter{Istio, Linkerd, Consul, Kuma, NSM, Nginx, AppMesh, Cilium, Traefik}
```
Comment (gyohuangxin) [inline server/models/adapter.go:23]: `Nighthawk` will be placed here when the meshery-nighthawk adapter is ready for use, right?

**Comment by l5io** [comment]
🚀 Preview for commit 44c36ac2ac4e01c4e67d45b4cabde4c8fd292ea4 at: https://65d597ef8875065a5dd4258c--meshery-docs-preview.netlify.app

File: server/models/adapter.go (hunk 1 above)
Comment (leecalcote) [reply server/models/adapter.go:23]: Good catch. My intention in this PR is to get Nighthawk entries fully ready, but to perhaps, leave it out its container deployment from the default list of containers to deploy. In other words, I missed this update. Thank you for pointing this area out. 

In this or other PR, we'll want to add the sub-chart for the new adapter. I think that you have write permission to this branch, so you are most welcome to make any changes (add commits) to this branch as you see fit, too. You won't be stepping on my toes. 👍

Comment (leecalcote) [inline server/models/adapter.go:23]:
```suggestion
var ListAvailableAdapters = []Adapter{Istio, Linkerd, Consul, Kuma, NSM, Nginx, AppMesh, Cilium, Traefik, Nighthawk}
```

Comment (leecalcote) [reply server/models/adapter.go]: I went ahead and added "Nighthawk" to the list. 👍

**Pull request merged**